src/StackAdmin/Azs.Update.Admin/readme.md

```diff
@@ -73,6 +73,16 @@ directive:
     set:
       default:
         script: -join("System.",(Get-AzLocation)[0].Name)
```

Comment:
can you please update the [0].Name to [0].Loccation in line 75? we have a new Az.Resources module that has this breaking change. Get-AzLocation cmdlet is from the module Az.Resources

Reply:
I made that commit on your branch. merging , thanks!

Reply:
sorry I just saw this. Thank you!